WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -277,13 +277,13 @@ private extension CardPresentPaymentsOnboardingUseCase {
         return wcPay.active && stripe.active
     }
 
-    func onlyStripeIsInstalled(wcPay: SystemPlugin?, stripe: SystemPlugin) -> Bool {
+    func onlyStripeIsInstalledAndActive(wcPay: SystemPlugin?, stripe: SystemPlugin) -> Bool {
```

Comment:
How does this test that the Stripe plugin is active?

Reply:
🤔 I guess it doesn't, but it also didn't check that it's installed, just assumes from the context where it's being called. But definitely could use a better name